client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -388,7 +395,10 @@ class BusinessDetails extends Component {
 									</Button>
 									{ hasInstallActivateError && (
 										<Button
-											onClick={ () => goToNextStep() }
+											onClick={ () => {
+												this.pushProfileUpdates();
```

Comment:
I think this will also need to be added to the "Continue" button.  In https://github.com/woocommerce/woocommerce-admin/issues/6718 the issue noted was anytime we switch over to the free extensions, selected options are not persisted.

Reply:
Right, I _did_ have it executing on that action but must have meddled with it before committing 🤭 . Fixed.